.changeset/shaggy-cheetahs-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/button': minor
```

Comment:
I'd probably call this a patch change. It's more a bug fix than a new feature

Reply:
What should the value be if it's a patch change? Is it just `'@leafygreen-ui/button': patch`?